include/deal.II/numerics/error_estimator.templates.h

```diff
@@ -169,9 +169,9 @@ namespace internal
        * Some more references to input data to the
        * KellyErrorEstimator::estimate() function.
        */
-      const typename FunctionMap<spacedim>::type *neumann_bc;
-      const ComponentMask                component_mask;
-      const Function<spacedim>                   *coefficients;
+      const typename FunctionMap<spacedim,number>::type *neumann_bc;
+      const ComponentMask                                component_mask;
+      const Function<spacedim>                          *coefficients;
```

Comment:
I admit to being completely naïve wrt. how this function works, but can one not now have complex `coefficients`? ie. Should it be `Function<spacedim>` -> `Function<spacedim,number>` here and below?